src/components/Layout.jsx

```diff
@@ -0,0 +1,19 @@
+import { Box } from '@mui/material';
+import Grid2 from '@mui/material/Grid2';
```

Comment:
Either use 

`import {Box, Grid2} from '@mui/material'`

or (preferred)

```
import Box from '@mui/material/Box'
import Grid2 from '@mui/material/Grid2'
```

but don't use a combination of both.